functions/Set-DbaAgentJobStep.ps1

```diff
@@ -372,9 +373,9 @@ function Set-DbaAgentJobStep {
                     #region job step options
 
                     # Execute
-                    if ($PSCmdlet.ShouldProcess($server, "Changing the job step '$StepName' for job '$j'")) {
+                    if ($PSCmdlet.ShouldProcess($server, "Committing changes for job step '$StepName' for job '$j'")) {
                         try {
-                            Write-Message -Message "Changing the job step '$StepName' for job '$j' on $server" -Level Verbose
```

Comment:
Verbose message not needed, as ShouldProcess always writes verbose output.

Reply:
it's true! I didnt realize this until years later. I'm going to merge nevertheless, since this is an important change